infra/nnfw/cmake/packages/GLib2.0Config.cmake

```diff
@@ -0,0 +1,43 @@
+function(_GLIB_2_0_import)
+  find_library(GLIB_LIBRARIES
+    NAMES glib-2.0
+    ONLY_CMAKE_FIND_ROOT_PATH)
+  
+  get_filename_component(GLIB_LIBRARY_DIR ${GLIB_LIBRARIES} DIRECTORY)
+  find_path(GLIBCONFIG_INCLUDE_DIR
+    NAMES glibconfig.h
+    PATHS ${GLIB_LIBRARY_DIR}
+    PATH_SUFFIXES glib-2.0/include
+    CMAKE_FIND_ROOT_PATH_BOTH)
+
+  find_path(GLIB_INCLUDE_DIR
+    NAMES glib.h
+    PATH_SUFFIXES glib-2.0
+    ONLY_CMAKE_FIND_ROOT_PATH)
```

Comment:
By toolchain file, `find_library` and `find_path` are check rooted directory only. Any reason to use `ONLY_CMAKE_FIND_ROOT_PATH` explicitly?

Reply:
IMO, this `find_path` and above `find_library` don't need `ONLY_CMAKE_FIND_ROOT_PATH`.